### ISSUE TITLE
legacy RSE are doing COBOL

### ISSUE THREAD

**Issue by mhagdorn** [the-teachingRSE-project/competencies#324]
is there really any research software that uses COBOL?

**Comment by CaptainSifff** [comment]
Good point....
any alternatives? Algol? Pascal? APL? Lisp?

**Comment by mhagdorn** [comment]
I think Pascal might be a better one.

**Comment by MakisH** [comment]
Remember that language standards also have versions and this could be an easy way out.

Legacy Fortran (meaning anything else than [modern Fortran](https://fortran-lang.org/), e.g., Fortran 77 and maybe Fortran 90, would be a clear candidate with many examples.

In the same category, I would also count (older) C and C++ pre-C++11 as legacy (but I don't know to what extent that could backfire). I definitely don't want to work with these nowadays if I have other options.

I also think it looks strange writing about COBOL in a research context.

**Comment by CaptainSifff** [comment]
hmmm.... so let's not state that we'Re so old, that we know these languages and instead do sth. like:
"legacy code written in Python-2 or pre-Fortran-77 code"?

**Comment by mhagdorn** [comment]
yes, that's much better. I'd say pre-fortran77 is truly ancient,  what about pre-Fortran95. F95 is when the language gained a lot useful features that made it quite modern.

**Comment by MakisH** [comment]
Maybe something like:

```
code written in older language standards, such as Python 2, Fortran 77, C++98, or other language standards considered deprecated by their communities.
```

**Comment by knarrff** [comment]
I am not against it, but putting Python 2 into the same basket as Fortran 77 seems a little strange to me. There are 30 years between these two, while one of them is only 16 years old. Python 3 was released 2008. To be comparable, we would have to use Fortran 2008 - but then of course that's still "new" to the Fortran community.
